fixtures/dom/src/components/Header.js

```diff
@@ -43,6 +52,15 @@ class Header extends React.Component {
           </span>
 
           <div className="header-controls">
+            <input
+              id="react_production"
+              type="checkbox"
+              checked={this.state.production}
+              onChange={this.handleProductionChange}
+            />
```

Comment:
Could you add a style to center this vertically with the label? maybe:

```css
.header__checkbox {
  vertical-align: middle;
}
```

Reply:
Yes, done.